wiki/en/Installation-for-iOS

```diff
@@ -0,0 +1,41 @@
+---
+layout: wiki
+title: "Installation for iOS"
+lang: "en"
+permalink: "/wiki/Installation-for-iOS"
+---
+
+{% include breadcrumb.html root="Using Jamulus" branch1="Getting Started" branch1-url="Getting-Started" %}
+
+
+# Installation for iOS
+
+Make sure you've already read the [Getting Started](Getting-Started) page.
+
+## Things to note about iOS
+Although Jamulus on iOS works quite well on new devices if everything is set up correctly, we strongly recommend only doing so if you have an audio interface and an USB-Ethernet adapter. To connect both devices to your iOS device, you should use something like the [Lightning to USB Camera Adapter](https://www.apple.com/uk/shop/product/MD821ZM/A/lightning-to-usb-camera-adapter) and a (powered USB-hub).
+If you have don't own a PC, we suggest you to buy a [Raspberry Pi](https://www.raspberrypi.org/){: target="_blank" rel="noopener noreferrer" } which is an inexpensive and small device that performs very well with Jamulus. iOS support is just a proof of concept and needs certain skills.
+
+## Install the iOS PoC (for experienced users only)
```

Comment:
```suggestion
## Install for iOS (for experienced users only)
```

Unless we want to say "proof of concept". The acronym isn't that common I don't think.